react/features/large-video/components/LargeVideo.web.js

```diff
@@ -70,6 +72,7 @@ export default class LargeVideo extends Component<*> {
                             muted = { true } />
                     </div>
                 </div>
+                <TranscriptionSubtitles />
```

Comment:
Yana mentioned something about transcriptions being enabled or disabled. Is the desire for it to be configurable?